customer-docs/user-manual.md

```diff
@@ -70,6 +70,8 @@ Streamliner uses configuration files which applied to templates to create DDL an
 2. `scripts` DDL and DML creation - Applies configuration properties to templates to produce DDL and DML scripts.  Templates are a collection of operations producing a data ingestion pipeline for a single table.
 
 ## Configuration
+Streamliner 5.x is compatible with Java 11.
```

Comment:
Move compatible java version to a separate heading.

Reply:
updated